crates/bevy_render/src/render_asset.rs

```diff
@@ -19,32 +19,34 @@ pub enum PrepareAssetError<E: Send + Sync + 'static> {
     RetryNextUpdate(E),
 }
 
+pub trait AssetUsages {
```

Comment:
I don't love the naming on this tbh :/

Reply:
Me neither. I can't call it `RenderAssetUsages` though, as that is already taken.

Reply:
Perhaps remove the enum, and make the RenderAssetUsages trait have two methods, `main_world() -> bool` and `render_world() -> bool`?

Reply:
It is now gone. :)